docs/advanced/dcrtime.md

```diff
@@ -0,0 +1,105 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Servers.svg" /> Dcrtime
+
+---
+
+Dcrtime is a timestamping application that allows users to store hashes of arbitrary data on the Decred blockchain. Inspired by Peter Todd's [opentimestamps](https://petertodd.org/2016/opentimestamps-announcement), dcrtime was originally built to store timestamped hashes of data from [Politeia](../../governance/politeia/overview), Decred's proposal system, on the Decred blockchain. 
+
+Although dcrtime was created for Politeia, it is expected to have generic utility as a timestamping service. Dcrtime is implemented as a client and server application, with all code open source and  available in the [dcrtime repo](https://github.com/decred/dcrtime). Decred also offers a free timestamping service through its public mainnet server. We expect this to be particularly useful in scenarios where transparency, accountability and time-ordering are of key importance, in either a public or private context (e.g. computer security, data integrity and various compliance contexts).
```

Comment:
double space "and  available"

Reply:
Could link "public mainnet server" to https://timestamp.decred.org/ ?

Reply:
Good catch. Good idea. Done